cluster_orchestrator/cluster-scheduler/calculation.py

```diff
@@ -16,12 +16,25 @@ def calculate(app, job):
 
 
 def constraint_based_scheduling(job, constraints):
+    filtered_active_nodes = []
     mongo_find_all_active_nodes()
```

Comment:
I just noticed this, probably coming from older commits... Is it just empty query? Since you're performing this query in "addons" case, can you remove this please?

Reply:
Good catch. Is removed.